app/components/govuk_component/section_break_component.rb

```diff
@@ -0,0 +1,40 @@
+class GovukComponent::SectionBreakComponent < GovukComponent::Base
+  SIZES = %w(m l xl).freeze
+
+  def initialize(visible: false, size: nil, classes: [], html_attributes: {})
+    super(classes: classes, html_attributes: html_attributes)
+
+    @visible = visible
+    @size    = size
+  end
+
+  def call
+    tag.hr(class: classes, **html_attributes)
+  end
+
+private
+
+  attr_reader :size, :visible
+
+  def default_classes
+    class_names(
+      "govuk-section-break",
+      size_class,
+      "govuk-section-break--visible" => visible?
+    ).split
+  end
+
+  def size_class
+    if size.blank?
+      ""
+    elsif size.in?(SIZES)
+      "govuk-section-break--#{size}"
+    else
+      raise ArgumentError, "invalid size #{size}, supported sizes are #{SIZES.to_sentence}"
```

Comment:
I'm not sure whether we should raise an error, or fallback and warn.

It looks like we raise errors elsewhere though, so I've maintained that behaviour for now.

Reply:
I think raising makes sense here, I'd want to find out immediately if there's an invalid size. Perhaps this is something that should be configurable though.